docs/pureDI/README.md

```diff
@@ -0,0 +1,43 @@
+# Integrating `Functional.CQS.AOP` Into Your Application When Using [Pure DI](http://blog.ploeh.dk/2014/06/10/pure-di/) (No Dependency Injection Container)
+
+Perhaps you have decided that [using a DI container is more trouble than it's worth for your simple application](http://blog.ploeh.dk/2012/11/06/WhentouseaDIContainer/)?
+
+A series of NuGet packages are provided that enable manual decoration of CQS handlers, all prefixed with `Functional.CQS.AOP.IoC.PureDI`.  Those packages are listed below.  As a matter of best practice, all `Functional.CQS.AOP.IoC.PureDI`-prefixed packages should only be installed in your application's [composition root](http://blog.ploeh.dk/2011/07/28/CompositionRoot/), as the decorators are an infrastructure concern and not a business domain concern.
+- `Functional.CQs.AOP.IoC.PureDI.MetricsCapturing` (decorators for posting performance metrics and unhandled errors)
+
+Decorators are applied to CQS handlers by fulfilling all decorator dependencies manually.  For example, here is an example query parameter object and query handler:
```

Comment:
Rather than using `example` twice, would it flow better to say, "For example, here is **a sample** query..."?